x-pack/plugins/security_solution/public/overview/components/host_overview/endpoint_overview/index.test.tsx

```diff
@@ -49,6 +53,16 @@ describe('EndpointOverview Component', () => {
         },
       }),
     };
+
+    useAgentStatusMock.mockReturnValue({
+      data: {
+        [endpointData.hostInfo?.metadata.agent.id!]: {
+          isolated: true,
+          pendingActions: {},
+          status: 'healthy',
+        },
+      },
+    });
   });
```

Comment:
Could you please check if Line 95-104 could be moved to x-pack/plugins/security_solution/public/common/components/endpoint/endpoint_agent_status/endpoint_agent_status.test.tsx?

Reply:
done 189f442df30cd132383eb355836af732ce0c982c